railties/lib/rails/generators/rails/plugin/USAGE

```diff
@@ -1,10 +1,19 @@
 Description:
-    The 'rails plugin new' command creates a skeleton for developing any
-    kind of Rails extension with ability to run tests using dummy Rails
-    application.
+    The `rails plugin new` command creates a Rails engine with ability
```

Comment:
It is not really an engine. It could be a railtie. That is why the description was generic talking about rails extensions.

Reply:
Oh, I see. It has been some time since I generated one so I'll take a look at the output for each option and amend. Thanks!